test/ioc/IocTest.java

```diff
@@ -0,0 +1,45 @@
+package ioc;
+
+import exceptions.ioc.IocDuplicatedKeyException;
+import exceptions.ioc.IocKeyNotFoundException;
+import exceptions.ioc.IocKeyNullPointerException;
+import exceptions.ioc.IocValueNullPointerException;
+import org.junit.Test;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class IocTest {
+    private Ioc ioc= Ioc.getInstance();
+
+    @Test
+    public void registerTest(){
```

Comment:
Use the following java convention for naming test methods: https://medium.com/@stefanovskyi/unit-test-naming-conventions-dd9208eadbea